src/meal.js

```diff
@@ -0,0 +1,185 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable radix */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable array-callback-return */
+
+async function listComments(itemId) {
```

Comment:
- [ ] It looks like here we could use ES6 best practices (in this case arrow function). 🐱